cloudbuild_CI.yaml

```diff
@@ -22,6 +22,8 @@ steps:
   - name: 'gcr.io/cloud-builders/docker'
     args:
       - 'build'
+      - '--build-arg'
+      - 'IMAGE_TAG=${COMMIT_SHA}'
```

Comment:
The downside of this is that even our released versions will not use the release tag, they will still use the commit SHA1.

I'm not sure there is a better option, though.